ground/gcs/share/taulabs/pfd/default/Compass.qml

```diff
@@ -44,7 +44,7 @@ Item {
             elementName: "homewaypoint-bearing"
             sceneSize: sceneItem.sceneSize
 
-            visible: HomeLocation.Set != 0
+            visible: HomeLocation.Set !== 0
```

Comment:
>... One of the statements in this comment is true...

^^^
I'm guessing that of the two statements in your comment, this is the true one? ;)